requirements.txt

```diff
@@ -1,2 +1,3 @@
-aioswitcher==3.4.2
+aioswitcher==4.0.0
+requests==2.32.3
```

Comment:
Can we make do with _aiohttp_ instead of adding _requests_?

Reply:
Yes. removed requests.